src/clj/rems/application/events.clj

```diff
@@ -63,7 +63,10 @@
 (s/defschema DraftSavedEvent
   (assoc EventBase
          :event/type (s/enum :application.event/draft-saved)
-         :application/field-values {s/Int s/Str}
+         :application/field-values {s/Int s/Str}))
+(s/defschema LicensesAcceptedEvent
+  (assoc EventBase
+         :event/type (s/enum :application.event/licenses-accepted)
          :application/accepted-licenses #{s/Int}))
```

Comment:
yeah it does look funny that we have both accepted-licenses and licenses-accepted